config/initializers/rswag_api.rb

```diff
@@ -0,0 +1,14 @@
+Rswag::Api.configure do |c|
```

Comment:
I'd not allow a single letter variable in the code base. It's config here.

Reply:
Agreed! I think this was generated automatically from a setup rake task provided by the gem.